src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -1234,6 +1269,32 @@ public void visitSource(String sourceName, SCMSourceObserver observer)
     } catch (HttpException e) {
       throw new AbortException(e.getMessage());
     }
+  }
+
+  /** {@inheritDoc} */
+  @Override
+  public void afterSave(@NonNull SCMNavigatorOwner owner) {
```

Comment:
this seems like a failed merged 😅 as you can see this is already master

https://github.com/jenkinsci/github-branch-source-plugin/blob/bd6d7f4c0b17b500c07608bc8eb6c47413f4ebca/src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java#L1569

Reply:
Thanks a lot for the help! I will fix that merge mistake asap! 😄

Reply:
@jetersen it is fixed now! Have a really great day! 🙂